pkg/views/target/set.go

```diff
@@ -302,3 +302,68 @@ func getFilePicker(name string, property apiclient.ProviderProviderTargetPropert
 		}),
 	}, value
 }
+
+func ParseJSON(data []byte, v interface{}) error {
+	if err := json.Unmarshal(data, v); err == nil {
+		return nil
+	}
+	return errors.New("input is not a valid JSON")
+}
+
+func ValidateProperty(targetManifest map[string]apiclient.ProviderProviderTargetProperty, target *TargetView) error {
```

Comment:
This function (and `ParseJSON`) can be inside `pkg/cmd/target/set.go` and be private.

Reply:
@Philip-21 I didn't mean all letters in lower case, only the first one